pkg/pillar/types/global.go

```diff
@@ -702,6 +705,9 @@ func NewConfigItemSpecMap() ConfigItemSpecMap {
 	configItemSpecMap.AddIntItem(NetworkTestTimeout, 15, 0, 0xFFFFFFFF)
 	configItemSpecMap.AddIntItem(NetworkSendTimeout, 120, 0, 0xFFFFFFFF)
 	configItemSpecMap.AddIntItem(Dom0MinDiskUsagePercent, 20, 20, 0xFFFFFFFF)
+	// Dom0DiskUsageMaxBytes - Default is 2GB, min is 512MB
```

Comment:
Why are these numbers immense? We need about 20M for logs so what is the remaining 1980M intended for?

Reply:
Set min to 100M for now.

Reply:
Thinking more.. there is no harm in making it 100MB.. User can always change it..
 Made the min value for the setting to be 100 MB ( 100*1024*1024 )